setup.py

```diff
@@ -41,7 +41,7 @@
         'cfgv>=1.0.0',
         'identify>=1.0.0',
         'nodeenv>=0.11.1',
-        'pyyaml>=4.2b4',
+        'pyyaml>=3.13',
```

Comment:
this should be fine to just leave unbounded -- pip will choose the latest version already as long as it isn't a pre-release